mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
Why do we need this ValueError ? Don't restrict it please.

2015-04-29 19:48 GMT+02:00 Eric Larson <notifications@github.com>:

> In mne/report.py
> <https://github.com/mne-tools/mne-python/pull/2039#discussion_r29361054>:
>
> > @@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
> >              warnings.warn('Could not import mayavi. Trying to render '
> >                            '`mayavi.core.scene.Scene` figure instances'
> >                            ' will throw an error.')
> > +        if np.isscalar(scale) and scale < 1.:
> > +            raise ValueError('scale, if float, must be >= 1 not %s'
> > +                             % scale)
>
> Why what?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2039/files#r29361054>.
>